docs-2.0/3.ngql-guide/4.variable-and-composite-queries/2.user-defined-variables.md

```diff
@@ -1,14 +1,31 @@
 # User-defined variables
 
-User-defined variables allows passing the result of one statement to another. User-defined variables are written as `$var_name`. The `var_name` consists of alphanumeric characters. Any other characters are not recommended currently.
+User-defined variables allows passing the result of one statement to another. User-defined variables are written as `$var_name`. The `var_name` consists of alphanumeric characters. Any other characters are not recommended.
```

Comment:
Any other characters are not recommended. -> Any other characters are not permitted.